lib/membrane_rtmp_plugin/rtmp_server.ex

```diff
@@ -15,7 +23,12 @@ defmodule Membrane.RTMP.Server do
           handler: ClientHandlerBehaviour.t(),
           port: :inet.port_number(),
           use_ssl?: boolean(),
-          name: atom() | nil
+          name: atom() | nil,
+          new_client_callback: (client_ref :: pid(),
+                                app :: String.t(),
+                                stream_key :: String.t() ->
+                                  any()),
+          client_timeout: non_neg_integer()
```

Comment:
In most plugins we use `Membrane.Time` for time handling, so it might be good to be consistent with that here as well:

```suggestion
          client_timeout: Membrane.Time.t()
```